Modules/Microsoft365DSC/Dependencies/Manifest.psd1

```diff
@@ -108,11 +108,6 @@
             ModuleName      = 'PnP.PowerShell'
             RequiredVersion = '1.12.0'
         },
-        @{
-            ModuleName      = 'PSDesiredStateConfiguration'
-            RequiredVersion = '1.1'
```

Comment:
Why removing this?

Reply:
This would be because of #5051, for instance I'm only not being affected by this like @ykuijs because due to my own needs I use a custom manifest which doesn't include it.

Reply:
This is because PSDesiredStateConfiguration isn't available in the PSGallery, but is installed by default with PSv5.1. When running the Update Dependencies cmdlet, this will never install because it is always present.....however when you have broken your environment the function cannot fix it because the module cannot be downloaded from the Gallery anyways.

Reply:
v1.1 that is

Reply:
Bottom line, are we okay with removing this from the manifest? @ricmestre

Reply:
@NikCharlebois Yes, let's remove this please